executor/aggregate.go

```diff
@@ -606,24 +601,30 @@ func (e *HashAggExec) parallelExec(ctx context.Context, chk *chunk.Chunk) error
 		e.prepare4ParallelExec(ctx)
 		e.prepared = true
 	}
-	for {
+
+	// gofail: var parallelHashAggError bool
```

Comment:
Why add this?

Reply:
It's a mistake to include these gofail comments.
They have been removed now.